filebeat/module/apache/error/test/sublevel.log

```diff
@@ -0,0 +1,2 @@
+[Wed Oct 20 19:20:59.121211 2021] [rewrite:trace3] [pid 121591:tid 140413273032448] mod_rewrite.c(470): [client 10.121.192.8:38350] 10.121.192.8 - - [api-dev.onintranet.com/sid#55a374e851c8][rid#7fb438083ac0/initial] applying pattern '^/bankimport/?(.*)$' to uri '/'
```

Comment:
nit: don't need to obfuscate this line? Does it contain any customer-specific data?

Reply:
Ok, I will change the urls.